### PR TITLE
redirect

### PR DESCRIPTION
**Pull request by mwernimont**
@jlucido2 all you need to hook up the redirect page?

### CODE REVIEW

File: glri-catalog-ui/src/main/webapp/redirect.jsp [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	<link href="./webjars/openlayers/2.13.1/theme/default/style.css" type="text/css" rel="stylesheet">
+	
+	
+</head>
+<body>
+    <div id="cover">
+	<div id="redirectInfo">
+	    <p>
+		The USGS GLRI Website and Science Explorer have been replaced. 
+		he new website is available at <a href="https://www.usgs.gov/centers/glri">https://www.usgs.gov/centers/glri</a>.
```
Comment (jlucido2) [inline glri-catalog-ui/src/main/webapp/redirect.jsp]: Should read 'The'

Comment (mwernimont) [reply glri-catalog-ui/src/main/webapp/redirect.jsp]: I'm confused, are you saying The isn't there or it needs ' ' around it?

Comment (jlucido2) [reply glri-catalog-ui/src/main/webapp/redirect.jsp]: Yep, was missing the 'T' in 'The'

**Comment by jlucido2** [comment]
Looks good!

**Pull request merged**